<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTreeView" name="treeView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::ActionsContextMenu</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QVTKOpenGLNativeWidget" name="vtkWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Add File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Options</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="actionItem_Options"/>
    <addaction name="actionStart_VR"/>
    <addaction name="actionStop_VR"/>
   </widget>
   <widget class="QMenu" name="menuFilters">
    <property name="title">
     <string>Filters</string>
    </property>
    <addaction name="actionClearFilters"/>
    <addaction name="actionClipFilter"/>
    <addaction name="actionShrinkFilter"/>
   </widget>
   <widget class="QMenu" name="menuFilter_All">
    <property name="title">
     <string>Filter All</string>
    </property>
    <addaction name="actionClearALL"/>
    <addaction name="actionClipALL"/>
    <addaction name="actionShrinkALL"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFilters"/>
   <addaction name="menuFilter_All"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_File"/>
   <addaction name="actionItem_Options"/>
   <addaction name="actionClearALL"/>
   <addaction name="separator"/>
   <addaction name="actionStart_VR"/>
   <addaction name="actionStop_VR"/>
  </widget>
  <action name="actionOpen_File">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/icons/fileopen.png</normaloff>:/Icons/icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionItem_Options">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/icons/options.png</normaloff>:/Icons/icons/options.png</iconset>
   </property>
   <property name="text">
    <string>Item Options</string>
   </property>
   <property name="toolTip">
    <string>Item Options</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionStart_VR">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/icons/startVR.png</normaloff>:/Icons/icons/startVR.png</iconset>
   </property>
   <property name="text">
    <string>Start VR</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionStop_VR">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/icons/stopVR.png</normaloff>:/Icons/icons/stopVR.png</iconset>
   </property>
   <property name="text">
    <string>Stop VR</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionShrinkFilter">
   <property name="text">
    <string>Shrink Filter</string>
   </property>
   <property name="toolTip">
    <string>Shrink Filter</string>
   </property>
  </action>
  <action name="actionClearFilters">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/icons/removeFilter.png</normaloff>:/Icons/icons/removeFilter.png</iconset>
   </property>
   <property name="text">
    <string>Clear Filters</string>
   </property>
   <property name="toolTip">
    <string>Clear Filters</string>
   </property>
  </action>
  <action name="actionClipFilter">
   <property name="text">
    <string>Clip Filter</string>
   </property>
   <property name="toolTip">
    <string>Clip Filter</string>
   </property>
  </action>
  <action name="actionShrinkALL">
   <property name="text">
    <string>ShrinkALL</string>
   </property>
   <property name="toolTip">
    <string>ShrinkALL</string>
   </property>
  </action>
  <action name="actionClipALL">
   <property name="text">
    <string>ClipALL</string>
   </property>
  </action>
  <action name="actionClearALL">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/icons/removeFilter.png</normaloff>:/Icons/icons/removeFilter.png</iconset>
   </property>
   <property name="text">
    <string>ClearALL</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QWidget</extends>
   <header>qvtkopenglnativewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
